app/src/org/commcare/network/CommcareRequestGenerator.java

```diff
@@ -131,7 +131,7 @@ public static CommcareRequestGenerator buildNoAuthGenerator() {
 
     public static HashMap getHeaders(String lastToken) {
         HashMap<String, String> headers = new HashMap<>();
-        headers.put("X-OpenRosa-Version", "2.1");
+        headers.put("X-OpenRosa-Version", "3.0");

```

Comment:
Is there a place where we keep track of what's included in these? I thought there was but now I can't track it down

Reply:
I assume you mean like a history of when it changed and what the change was for? If so, I don't know of one existing.

Reply:
Aha, found it at the bottom of
https://github.com/dimagi/commcare-core/wiki/SyncAndRestoreAPI

which defintely doesn't seem like the right place. I'll move it over

Reply:
updated  here: https://github.com/dimagi/commcare-core/wiki/OpenRosaRequest

does this sound right?